### ISSUE TITLE
Check if good UX is possible with less than 16 physical cores

### ISSUE THREAD

**Issue by berendt** [osism/cloud-in-a-box#27]
In the current cloud-in-a-box systems we have 1 socket with 16 physical cores. It should be checked if we still have a good UX when we recommend less than 16 physical cores.

**Comment by berendt** [comment]
Memory footprint with all services up & running: 

```
dragon@manager:~$ free -m
               total        used        free      shared  buff/cache   available
Mem:           64163       11537       49878          71        2747       51817
Swap:           8191           0        8191
```

**Comment by berendt** [comment]
1 socket with 4 cores + 32 GByte memory works good.

**Comment by berendt** [comment]
Fixed with 4c88ddd1d6464d48cd15c24176064044d7e7ece5.
